frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -793,6 +793,59 @@
             utilities.sendRequest(parameters);
         };
 
+        vm.showRemainingSubmissions = function() {
+            var parameters = {};
+            vm.remainingSubmissions = {};
+            vm.remainingTime = {};
+            vm.showClock = false;
+            vm.showSubmissionNumbers = false;
+            parameters.url = "jobs/"+ vm.challengeId + "/phases/"+ vm.phaseId + "/remaining-submissions/";
```

Comment:
I will suggest making a config file and getting details from the object of the config file.